### PR TITLE
WFP-769 updated deployment to new api version

### PR DESCRIPTION
**Pull request by carlov20**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,11 +6,11 @@ This directory contains the manifest files needed to deploy a standard maintenan
 This directory also contains the maintenance page HTML file, along with a DockerFile to compile an image.
 
 ## Deploying the page
-Within the `kubectl_deploy` directory, there are 2 simple manifest files that make up the deployment, `deploy.yaml` and `service.yaml`.
```
Comment (ushkarev) [inline README.md:9]: you’ll need to rename the folder to match this

Comment (carlov20) [reply README.md:9]: the folder is already named `kubect_deploy`, happy to update the folder to be `kubectl_deploy` and not update the readme?

Comment (ushkarev) [reply README.md:9]: hahaha i'm blind! i somehow thought your rename was in the opposite direction. ignore me :)

Comment (poornima-krishnasamy) [reply README.md:9]: Thanks @carlov20  for the PR. I think it is better to update the folder to `kubectl_deploy` so it doesnot look like a typo.

**Pull request closed without merging**